osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -331,9 +321,6 @@ private void updateLogoState(MenuState lastState = MenuState.Initial)
                     {
                         logoTracking = false;
 
-                        hideOverlaysOnEnter.Value = true;
```

Comment:
moving this out has caused a regression in the menu animation (timings have changed)